scripts/location_utils.py

```diff
@@ -36,6 +36,12 @@ def get_postcode_from_questions(form_questions) -> str:
                     postcode = raw_postcode.strip().replace(" ", "").upper()
                     return postcode
 
+            else:
+                print(
+                    "The key 'yEmHpp' for Project information form not found"
+                )
+                return "Not found"
```

Comment:
Hopefully there is no postcode named "Not found" 😉

Reply:
Hope not 😆 but it shouldn't happen in production anyway